extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -142,16 +156,27 @@ export class Bind {
    * @private
    */
   initialize_() {
+    dev().fine(TAG, 'Scanning DOM for bindings...');
+
     this.scanPromise_ = this.scanBody_(this.ampdoc.getBody());
     this.scanPromise_.then(results => {
       const {boundElements, bindings, expressionToElements} = results;
-
       this.boundElements_ = boundElements;
-
       Object.assign(this.expressionToElements_, expressionToElements);
+      dev().fine(TAG, `Scanned ${bindings.length} bindings from ` +
+          `${boundElements.length} elements.`);
 
-      this.evaluator_ = new BindEvaluator();
-      const parseErrors = this.evaluator_.setBindings(bindings);
+      // Parse on web worker if experiment is enabled.
+      if (this.workerExperimentEnabled_) {
```

Comment:
Are you making this an experiment to not disrupt your other development, for easy benchmarking, or is there a different reason?